### DISCUSSION TITLE
How to Chakra UI styling root, instead of body, I want to attach the style to DOM that place I render React Component

### DISCUSSION THREAD

**Discussion by hardi-dev** [chakra-ui/chakra-ui#5147] [Q&A]
Hello everyone, 

I have a problem, i creating a chrome extension with react + chakra ui.

but chakra ui affecting the style of existing website that i visit whenever chrome extension that i develop is active.

i found on documentation to change css root var, but not for styling part

is there a way to change chakra ui root styling, instead of affecting the body, i want it just affecting the dom that i create to render React component.

thanks

**Comment by FeeLeeeD** [comment]
Hey, @hardi-dev! Have you solved your issue?

**Reply by ChinmayGopal931** [reply]
Hey, Chakra Ui seems to be effecting the useContext hook I use to change the desktop background I use. How can I stop chakra from doing that? or Maybe how do I change th ebackgorund using chakra?
https://stackoverflow.com/questions/72889799/how-to-usecontect-to-change-the-background-when-using-chakraui
